CHANGELOG.asciidoc

```diff
@@ -34,11 +34,10 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Support for `transaction.marks` {pull}430[430]
 - Support for uploading sourcemaps {pull}302[302].
 - Support for sourcemap mapping on incoming frontend requests {pull}381[381], {pull}502[502]
-- Support for `transaction.marks` {pull}430[430].
```

Comment:
shouldn't this be the one that stays since it's newer?

Reply:
maybe :) what's the difference?

Reply:
updated now